docs/README.md

```diff
@@ -1,3 +1,37 @@
+# Installation Prerequisites
+
+1. [Install the latest docker](https://docs.docker.com/get-docker/) and you may want to install [docker desktop - latest](https://docs.docker.com/desktop/)
+2. [Install python 3.9.5 (latest) and](https://www.python.org/downloads/) [configure your environment](https://docs.python.org/3/tutorial/venv.html) to get data-platform source environment ready.
+3. Install Visual Studio Code (python packages & typescript) <TODO: add .vscode>
+4. [Installing nvm](https://github.com/nvm-sh/nvm#installing-and-updating) and then do:
```

Comment:
Other steps have `Install` as opposed to `Installing`

Reply:
yeah i'll fix when i update the python code